docs/source/install.rst

```diff
@@ -54,56 +54,18 @@ The instructions above install a CPU-only version of SCICO. To install a version
 
    ::
 
-      pip install --upgrade "jaxlib==0.1.70+cuda110" -f https://storage.googleapis.com/jax-releases/jax_releases.html
+      pip install --upgrade "jaxlib==0.1.75+cuda110" -f https://storage.googleapis.com/jax-releases/jax_releases.html
 
 
 
 Additional Dependencies
 -----------------------
 
-We use the `ASTRA Toolbox <https://www.astra-toolbox.com/>`_ for tomographic projectors. We currently require the development version of ASTRA, as suggested by the package maintainers.
-
-The development version of ASTRA can be installed using conda:
-
-::
-
-   conda install -c astra-toolbox/label/dev astra-toolbox
-
-Alternatively, it can be `built from source <https://www.astra-toolbox.com/docs/install.html#for-python>`_.
-
-We also support the `Super-Voxel Model-Based Iterative Reconstruction <https://svmbir.readthedocs.io/en/latest/>`_ package as an alternative tomographic projector. Since this package can be installed via ``pip``, it is
```

Comment:
If we mention `astra`, perhaps `svmbir` also deserves an explicit mention, despite being included in the requirements file?